<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LV1Zyfrz</class>
 <widget class="QMainWindow" name="LV1Zyfrz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="txtInput">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>3</y>
      <width>421</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lstCypher">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>256</width>
      <height>71</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <item>
     <property name="text">
      <string>Atbash</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Affine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Beufort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ceasir</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fibbanacci Ceasir</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Playfair</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vigenere</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ExicuteButton">
    <property name="geometry">
     <rect>
      <x>631</x>
      <y>380</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>exicute</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdoDecrypt">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>340</y>
      <width>103</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>decrypt</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdoEncrypt">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>103</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>encrypt</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lstSubstitution">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <item>
     <property name="text">
      <string>1:Arrow Alphabit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2:Egyptian</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3:Morse Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4:Playing Cards</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5:Greek</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6:Hebrew</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7:Alchemical Symbles</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8:Ancient Symbles</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9:Wingdings</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10:Unique</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11:Weather Emojis</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12:Chess Peices</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13:Emoti</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14:Fruits</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15:Playing Cards 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16:Cars</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17:Dominoes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18:majong</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19:sumerian</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="chkSubstitution">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>582</x>
      <y>230</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>substitution </string>
    </property>
   </widget>
   <widget class="QToolButton" name="helpButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>490</y>
      <width>27</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtOutput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>571</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
